functions/venue.js

```diff
@@ -828,7 +827,7 @@ exports.updateVenue_v2 = functions.https.onCall(async (data, context) => {
   const venuesRef = await admin
     .firestore()
     .collection("venues")
-    .where("slug", "==", data.slug)
+    // .where("slug", "==", data.slug)
     .where("worldId", "==", data.worldId)
```

Comment:
This will not work because it will always return the same venue doc for all of the venues in that world, because of the logic below `docs[0]`

Reply:
made a mistake by committing this change, thanks for noticing it! 06cbf3a